contracts/src/interfaces/Teleporter/IAvalancheICTTRouterFixedFees.sol

```diff
@@ -19,6 +24,10 @@ interface IAvalancheICTTRouterFixedFees is IAvalancheICTTRouter {
     error AvalancheICTTRouterFixedFees__CumulatedFeesExceed100Percent(
         uint256 primaryRelayerFeeBips, uint256 secondaryRelayerFeeBips
     );
+    error AvalancheICTTRouterFixedFees__RelayerFeesTooLow(
+        uint256 primaryRelayerFee, uint256 secondaryRelayerFee, MinBridgeFees minBridgeFees
+    );
```

Comment:
I think this will be pretty annoying to debug. Better to use 4 `uint256` directly

Reply:
Not so sure about that. Seems clean enough for me to debug.